pkgs/applications/networking/browsers/chromium/common.nix

```diff
@@ -479,7 +480,7 @@ let
       # libpci (from pciutils) is needed by dlopen in angle/src/gpu_info_util/SystemInfo_libpci.cpp
       chromiumBinary="$libExecPath/$packageName"
       origRpath="$(patchelf --print-rpath "$chromiumBinary")"
-      patchelf --set-rpath "${lib.makeLibraryPath [ libGL vulkan-loader pciutils ]}:$origRpath" "$chromiumBinary"
+      patchelf --set-rpath "${lib.makeLibraryPath [ libGL libglvnd vulkan-loader pciutils ]}:$origRpath" "$chromiumBinary"
```

Comment:
Isn't libGL already libglvnd on linux?

Reply:
Yes, `libGL/lib` actually is just a symlink to `libglvnd/lib`. I really don't know why but this single change fixes electron-apps in Wayland for me.
Without this change, `electron` contains just `libGL` in its runpath and then the Wayland backend fails to find and load `libEGL.so.1`. Adding `libglvnd` (which is the exact same path `libGL` links to) to the runpath somehow fixes the issue and I have no idea why.
I guess some dynamic loader somewhere inside `electron` itself doesn't respect symlinks when searching the runpath for `libEGL.so.1`?